src/components/Trade/TradeConfirm/AssetToAsset.tsx

```diff
@@ -55,8 +55,8 @@ export const AssetToAsset = ({
   }
 
   return (
-    <Flex width='full' justifyContent='space-between' {...rest}>
-      <Box flex={1}>
+    <Flex width='full' justifyContent='space-between' alignItems='stretch' {...rest}>
+      <Box flex={1} maxWidth={`calc(50% - ${boxSize} / 2)`}>
```

Comment:
@reallybeard we need to limit the width to make the asset value line-breaking, wdyt about this?